src/molecule/test/functional/test_command.py

```diff
@@ -287,6 +296,20 @@ def test_command_test(scenario_to_test, with_scenario, scenario_name, driver_nam
     run_test(driver_name, scenario_name)
 
 
+@pytest.mark.serial
+@pytest.mark.parametrize(
+    "scenario_to_test, driver_name, scenario_name, platform_name",
+    [
+        ("driver/delegated", "delegated", "default", "instance"),
+    ],
+    indirect=["scenario_to_test", "driver_name", "scenario_name", "platform_name"],
+)
+def test_command_test_with_platform_name(
+    scenario_to_test, with_scenario, scenario_name, driver_name, platform_name
+):
+    run_test_with_platform_name(driver_name, platform_name, scenario_name)
```

Comment:
Since this function is used as a name, I'd say that it should arguably be implemented right here, not in conftest. Conftest exists for test helpers, not test logic. Would you move it, please?

Reply:
Thank you. I agree with you.

This function is moved from `conftest` into `test_command`.

But I noticed there are 2 function still in `conftest`, [run_test and verify](https://github.com/ansible-community/molecule/blob/4f8fd055ab183a1928b3460ccb39e94ac6162ebf/src/molecule/test/functional/test_command.py#L31-L32)

Do we need to move them either?